x/staking/README.md

```diff
@@ -83,7 +83,7 @@ UnbondingID stores the ID of the latest unbonding operation. It enables to creat
 
 * UnbondingID: `0x37 -> uint64`
 
-## Params
+## Parameters
```

Comment:
I don't understand the relation with the PR title and this fix 😅 
If we modify that, can you update the ToC link too?

Reply:
the toc was pointing to parameters, but the title was params so the link wasn't working. this fixed it

Reply:
Wdym, the toc was pointing to params: https://github.com/cosmos/cosmos-sdk/pull/13745/files/30648a50f82b41932fcdf67121e2223d08bcdff6#diff-8cae4b5d28bad715411b21d035d60df66c05405a5b166c24b1eb5000998e16d6R26

Reply:
Reverting this in another doc PR then: https://github.com/cosmos/cosmos-sdk/commit/a428f39e297c8d5ad56da15ca0a9d4186b83960a

Reply:
please don't revert it works now